jp2_pc/Source/Trespass/supportfn.cpp

```diff
@@ -936,14 +936,13 @@ CCamera* pcamGetCamera()
 
 void SetupGameScreen()
 {
-    POINT clientSize = GetCurrentClientSize();
-	
-    int             iWidth = clientSize.x;
-    int             iHeight = clientSize.y;
+    int             iWidth;
+    int             iHeight;
     BOOL            bSystemMem;
     RECT            rc;
     int             iGore;
 
+    bGetDimensions(iWidth, iHeight);
 	
```

Comment:
Using the client rectangle size here caused the window in windowed mode to gradually shrink when alt-tabbing. In exclusive fullscreen mode, it caused the game to resize to the regular desktop resolution upon a level transition.
We go back to using the configured screen size.